modules/apps/frontend-editor/frontend-editor-alloyeditor-web/src/main/resources/content/Language.properties

```diff
@@ -0,0 +1 @@
+invalid-drag-and-drop-action=Invalid drag and drop action.
```

Comment:
@jonmak08, I'm unable to get this new language key working.

Reply:
I'll take a look.

Reply:
I think because this is a contributor and not a protlet, we need to do something different to include the language keys.  I'll do some more investigation about this.